sematic/versions.py

```diff
@@ -9,7 +9,7 @@
 # the sdk. Should be bumped any time a release is made. Should be set
 # to whatever is the version after the most recent one in changelog.md,
 # as well as the version for the sematic wheel in wheel_version.bzl
-CURRENT_VERSION = (0, 21, 0)
```

Comment:
Even if client versions < 0.20.0 are supported, version 0.20.0 is not supported.
So `MIN_CLIENT_SERVER_SUPPORTS` should be upgraded to 0.20.1 as well.